server/services/v1/auth_provider.go

```diff
@@ -280,7 +301,13 @@ func getAccessTokenUsingRefreshToken(req *api.GetAccessTokenRequest, a *Auth0) (
 	if err != nil {
 		return nil, api.Errorf(api.Code_INTERNAL, "Failed to get access token: reason = %s", err.Error())
 	}
-	defer resp.Body.Close()
```

Comment:
```
tr := &http.Transport{TLSClientConfig: &tls.Config{MinVersion: tls.VersionTLS12}}
ctxClient := context.WithValue(context.Background(), oauth2.HTTPClient, &http.Client{Transport: tr, Timeout: tokenRequestTimeout})
t := &oauth2.Token{AccessToken: token}
ocfg1 := &oauth2.Config{Endpoint: oauth2.Endpoint{TokenURL: tokenURL}}
token, err = ocfg1.TokenSource(ctxClient, t).Token()
```

Reply:
I will handle this change in different PR.